Source/bmalloc/libpas/src/libpas/pas_page_malloc.c

```diff
@@ -54,6 +54,8 @@ bool pas_page_malloc_decommit_zero_fill = false;
 
 #if PAS_OS(DARWIN)
 #define PAS_VM_TAG VM_MAKE_TAG(VM_MEMORY_TCMALLOC)
+#elif BPLATFORM(PLAYSTATION) & defined(VM_MAKE_TAG)
```

Comment:
Use PAS_OS etc. instead since this is libpas, not bmalloc

Reply:
Thank you. This should be `PAS_PLATFORM(PLAYSTATION)`.